README.md

```diff
@@ -4,38 +4,36 @@
 
 ![Powered by OpenShift](/docs/img/powered_by_openshift.png)
 
-- [What is Odo?](#what-is-odo)
-- [Why did we create Odo?](#why-did-we-create-odo)
-- [Who is Odo for?](#who-is-odo-for)
-- [Setup and installation](#setup-and-installation)
-- [Deploying an application using Odo](#deploying-a-nodejs-application-using-odo)
-- [Additional documentation](#additional-documentation)
+- [What is Odo?](#odo)
+- [Features](#features)
+- [Setup and Installation](#setup-and-installation)
+- [Deploying an Application using Odo](#deploying-a-nodejs-application-using-odo)
+- [Additional Documentation](#additional-documentation)
 - [Community, Discussion, Contribution and Support](#community-discussion-contribution-and-support)
 - [Glossary](#glossary)
 - [CLI Structure](#cli-structure)
 
 
-## What is Odo?
+## Odo
 
 OpenShift Do (Odo) is a CLI tool for developers who are writing, building, and deploying applications on OpenShift. With Odo, developers get an opinionated CLI tool that supports fast, iterative development which abstracts away Kubernetes and OpenShift concepts, thus allowing them to focus on what's most important to them: code.
 
+Odo was created to improve the developer experience with OpenShift. Existing tools such as `oc` are more operations-focused and requires a depp-understanding of Kubernetes and OpenShift concepts. Odo is designed to be simple and concise so you may focus on coding rather than how to deploy your application. Since Odo can build and deploy your code to your cluster immediately after you save you changes, you benefit from instant feedback and can thus validate your changes in real-time. 
```

Comment:
Typo: depp->deep. Also I don't think that `deep understanding` is hyphenated.

Reply:
I would still mention `odo` core concepts here (project, application, components).